tests/ConnectionTest.php

```diff
@@ -196,14 +196,51 @@ public static function dataConnectionConfig(): Generator
     public function testConnectionConfig(string $expectedUri, string $expectedDatabaseName, array $config): void
     {
         $connection = new Connection($config);
-        $client     = $connection->getMongoClient();
+        $client     = $connection->getClient();
 
         $this->assertSame($expectedUri, (string) $client);
         $this->assertSame($expectedDatabaseName, $connection->getMongoDB()->getDatabaseName());
         $this->assertSame('foo', $connection->getCollection('foo')->getCollectionName());
         $this->assertSame('foo', $connection->table('foo')->raw()->getCollectionName());
     }
 
+    public function testLegacyGetMongoClient(): void
+    {
+        $connection = DB::connection('mongodb');
+        $expected = $connection->getClient();
+
+        $this->assertSame($expected, $connection->getMongoClient());
+    }
+
+    public function testLegacyGetMongoDB(): void
```

Comment:
Do these legacy tests need to expect a deprecation message?

Reply:
Something catches deprecation messages and hides them. I couldn't add an assertion.